libdevcore/CommonIO.cpp

```diff
@@ -126,6 +129,29 @@ std::string dev::getPassword(std::string const& _prompt)
 	std::getline(cin, ret);
 	return ret;
 #else
-	return getpass(_prompt.c_str());
+	struct termios oflags;
+	struct termios nflags;
+	char password[128];
```

Comment:
Have a few friends who use >128chars for their ssh passwords. Can we bump this to 256?